### PR TITLE
PLAT-139208:  Panels: Fix console error - React does not recognize the `controlsRef` and `controlsMeasurements`

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
qa-sample for `pattern-video-layer` had an console error when clicking on the `menu` button from mediaControls. 

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
I added `delete rest.controlsRef` and `delete rest.controlsMeasurements` in Panels/Viewport.js in order to remove them from the DOM.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-139208

### Comments

Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact moonstone module, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+
+- `moonstone/Viewport` to not pass `controlsRef` and `controlsMeasurements` to ui/ViewManager
```
Comment (MikyungKim) [inline CHANGELOG.md]: I think we don't have to write a changelog for this. :)

Comment (adrian-cocoara-lgp) [reply CHANGELOG.md]: 👍

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**